let User,CustomErrorHandler;_0c5‍.x([["default",()=>_0c5‍.o]]);_0c5‍.w("../../models",[["User",["User"],function(v){User=v}]]);_0c5‍.w("../../services",[["CustomErrorHandler",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);


const userController = {
    async getUserDetails(req, res, next) {
        try {
            const user = await User.findOne({ _id: req.user._id }).select('-password -updatedAt -__v');
            if (!user) {
                return next(CustomErrorHandler.notFound());
            }
            res.json(user);
        } catch (err) {
            return next(err);
        }
    }
};

_0c5‍.d(userController);